tutorials/run-meteor-on-google-app-engine.md

```diff
@@ -142,20 +142,33 @@ running the following command:
 
 1. Replace the contents of the `Dockerfile` file with the following:
 
-        FROM gcr.io/google_appengine/nodejs
+        FROM launcher.gcr.io/google/nodejs
+        RUN install_node v4.6.2
         COPY . /app/
         RUN (cd programs/server && npm install --unsafe-perm)
         CMD node main.js
 
     The custom `Dockerfile` is required in order to properly build the Meteor
     app in production.
 
-1. Add the following to the generated `app.yaml` file:
+1. Replace the contents of the generated `app.yaml` file:
```

Comment:
Please make a follow-on commit with the additional changes you requested to `app.yaml`.